shopgate.php

```diff
@@ -171,6 +173,8 @@ public function install()
             'header',
             'adminOrder',
             'updateOrderStatus',
+            'updateQuantity',
```

Comment:
Hi @OliverMuellerShopgate,

Exactly, the `$module` parameter will be an instance of ShopGate. :)
And yes, the documentation you found is what I was talking about. These upgrade are usefull when the module database is updated between two versions, or when hooks need to be registered or unregistered.

If you want to test your upgrade file, just install your module from the Backoffice (which will be downloaded from Addons), and then upload your upgraded version. If everything goes right, you will see at the top of the page that an upgrade file has been successfully applied.